src/AppBundle/Command/AddUserCommand.php

```diff
@@ -0,0 +1,280 @@
+
+            $email = $console->ask($input, $output, $question);
+            $input->setArgument('email', $email);
+        } else {
+            $output->writeln(' > <info>Email</info>: '.$email);
+        }
+    }
+
+    /**
+     * This method is executed after interact() and initialize(). It usually
+     * contains the logic to execute to complete this command task.
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $startTime = microtime(true);
+
+        $username = $input->getArgument('username');
+        $plainPassword = $input->getArgument('password');
+        $email = $input->getArgument('email');
+        $isAdmin = $input->getOption('is-admin');
+
+        // first check if a user with the same username already exists
+        $existingUser = $this->entityManager->getRepository('AppBundle:User')->findOneBy(array('username' => $username));
+
+        if (null !== $existingUser) {
+            throw new \RuntimeException(sprintf('There is already a user registered with the "%s" username.', $username));
+        }
+
+        // create the user and encode its password
+        $user = new User();
+        $user->setUsername($username);
+        $user->setEmail($email);
+        $user->setRoles(array($isAdmin ? 'ROLE_ADMIN' : 'ROLE_USER'));
+
+        // See http://symfony.com/doc/current/book/security.html#security-encoding-password
+        $encoder = $this->getContainer()->get('security.password_encoder');
+        $encodedPassword = $encoder->encodePassword($user, $plainPassword);
+        $user->setPassword($encodedPassword);
+
+        $this->entityManager->persist($user);
+        $this->entityManager->flush();
+
+        $output->writeln('');
+        $output->writeln(sprintf('[OK] %s was successfully created: %s (%s)', $isAdmin ? 'Administrator user' : 'User', $user->getUsername(), $user->getEmail()));
+
+        if ($output->isVerbose()) {
+            $finishTime = microtime(true);
+            $elapsedTime = $finishTime - $startTime;
+
+            $output->writeln(sprintf('[INFO] New user database id: %d / Elapsed time: %.2f ms', $user->getId(), $elapsedTime*1000));
+        }
+    }
+
+    /**
+     * This internal method should be private, but it's declared as public to
+     * maintain PHP 5.3 compatibility when using it in a callback.
+     *
+     * @internal
+     */
+    public function passwordValidator($plainPassword)
+    {
+        if (empty($plainPassword)) {
+            throw new \Exception('The password can not be empty');
+        }
+
+        if (strlen(trim($plainPassword)) < 6) {
+            throw new \Exception('The password must be at least 6 characters long');
+        }
+
+        return $plainPassword;
+    }
+
+    /**
+     * This internal method should be private, but it's declared as public to
+     * maintain PHP 5.3 compatibility when using it in a callback.
+     *
+     * @internal
+     */
```

Comment:
__[development][code_sniffer]__
* Missing @return tag in function comment

<!-- Comment by [SideCI](https://sideci.com) -->